apps/cowswap-frontend/src/common/pure/NetworksList/index.tsx

```diff
@@ -27,7 +27,13 @@ export function NetworksList(props: NetworksListProps) {
         const isActive = targetChainId === currentChainId
 
         const rowContent = (
-          <styledEl.FlyoutRow key={targetChainId} onClick={() => onSelectChain(targetChainId)} active={isActive}>
+          <styledEl.FlyoutRow
+            className="network-item"
```

Comment:
I think is not great to add classNames just for the E2E

Yes, we have a mess there, but we could see if there's way to do it more semantically. 
cypress guys recommend using `data`


they recommend for example:

`cy.get('[data-testid="submit"]').click()`

I don't love it because, because IMO, e2e should be defined as from the eyes of a user. "Click the submit button saying Print Invoice" , but i guess they know better


https://docs.cypress.io/guides/references/best-practices